simulation/examples/fastest_time_given_distance.py

```diff
@@ -44,13 +43,25 @@ def main():
     with open(settings_directory / "initial_conditions.json") as f:
         args = json.load(f)
 
-    initialSimulationConditions = SimulationState(args)
+    initialSimulationConditions = simulationState.SimulationState(args)
 
-    simulation_model = TimeSimulation(initialSimulationConditions, race_type="ASC")
+    golang = True
+
```

Comment:
Can we make this an argument that the user passes as they call the file? So we would be able to call it as 

python fastest_time_given_distance.py -goland = True

Reply:
Yep! I've also added a "-help" option to display a list of valid commands as I'm assuming we're going to be adding more eventually.